zerver/lib/webhooks/git.py

```diff
@@ -1,4 +1,4 @@
-from six import text_type
+from typing import Text
```

Comment:
No, you should `amend` your commit and then do the force push. I will public here url to our tutorial how to do it.